<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QWidget" name="Registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>授权码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AccreditCode_LineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>270</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入授权码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="MachineCode_LineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>270</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>请将该机器码发给软件供应商以获取授权码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>机器码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="state_Label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnTurnBack">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>400</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>返 回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Accredit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>授 权</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
